examples/custom-service.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "Custom Service"
```

Comment:
Thanks for taking on this one, it's tricky! I think we might want to use something like a combined axum and serenity app for this, since currently it's the most common usecase. If you'd like to give that one a go, it should be a lot like the thread I linked in linear. If you don't have time for this, though, that's perfectly fine too. :slightly_smiling_face: